### DISCUSSION TITLE
GET repo doesn't return delete_branch_on_merge key

### DISCUSSION THREAD

**Discussion by stevecorey** [community/community#24359] [Enterprise]
<p>Using GitHub Enterprise Server 2.20.15 and PowerShell to GET /repos/{owner}/{repo}<br>
But the response doesn’t contain the delete_branch_on_merge key.<br>
The PowerShell command I used:<br>
$repoInfo = Invoke-RestMethod -Uri “<a href="https://github.DOMAIN.com/api/v3/repos/ORG/REPO?access_token=TOKEN" rel="noopener nofollow ugc">https://github.DOMAIN.com/api/v3/repos/ORG/REPO?access_token=TOKEN</a>” -Headers @{‘Accept’ = ‘application/vnd.github.v3+json’ }<br>
Other keys that are specified in the API documentation are returned, like allow_squash_merge and allow_merge_commit<br>
I have noted that delete_branch_on_merge is not settable until v2.21 <a href="https://enterprise.github.com/releases/2.21.0/notes" rel="noopener nofollow ugc">https://enterprise.github.com/releases/2.21.0/notes</a><br>
But is there a way to get it in v2.20.15?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Hello <a class="mention" href="/stevecorey">@stevecorey</a> and welcome to the community.</p>
<p>That field isn’t available on GHES v2.20.15. It is available on GHES v2.21 though.</p>
<p>Let us know if you have more questions.</p>
